Documentation/SwiftGenKit Contexts/Plist.md

```diff
@@ -38,16 +38,21 @@ files:
       User Integer: 5
     metadata:
       properties:
-      - name: User Ambiguous Integer
-        type: Bool
-      - name: User Boolean
-        type: Bool
-      - name: User Date
-        type: Date
-      - name: User Float
-        type: Double
-      - name: User Integer
-        type: Int
+        User Ambiguous Integer:
+          name: User Ambiguous Integer
```

Comment:
I'll have to check my Stencil-foo a bit, but I wonder if it's necessary to repeat the name as a key here.

The main usages I see to navigate this metadata structure is:

1) Using a `{% for key, value in metadata %}`: in that case, the name of the key is already parsed by Stencil into the `key` variable during the iteration
2) Using the `metadata[key]` upcoming Stencil subscript syntax, in which case that supposes we already know the key

Maybe I'm missing a use case in a template where we'd be iterating over the metadata without extracting the key during iteration (like in `{% for k,v in … %}`), or a use case using filters or whatnot to navigate this structure where the `name` key would be useful? But otherwise we could as well remove it and only keep `type` (and `properties` if it's a nested dictionary)